src/Tree.tsx

```diff
@@ -744,6 +744,9 @@ class Tree extends React.Component<TreeProps, TreeState> {
    * We have to use `cloneElement` to pass `key`.
    */
   renderTreeNode = (child, index, level = 0) => {
+    if (!React.isValidElement(child)) {
+      return null;
```

Comment:
should be `return child`?

Reply:
If an element is invalid, I think there's no difference between `child` and `null`.` return child` should be also fine.